tests/test_interface.rb

```diff
@@ -1344,6 +1351,7 @@ def test_interface_ipv4_all_interfaces
 
     # pre-configure
     begin
+      interface_ethernet_default(interfaces[1])
```

Comment:
* Line 1355 is a pre-config dependency for several tests that follow.
* The pre-config failed when I ran the full test because some previous, unrelated test left some vni configs on that interface that interfered with setting channel-group. 
* Notice that it's a different interface than the one that's used for all other tests in this massive test file.

Since this is the only place it's used it's better to just do it here than in `setup`, which would slow down every test case.